server/WFCORE-6728-reload-to-stability-level.adoc

```diff
@@ -0,0 +1,155 @@
+---
+categories:
+  - core
+# Add any category for this proposal
+# if missing, add it to _data/widfly-categories and use its id
+---
+= Ability to reload a server to a different stability level
+:author:            Kabir Khan
+:email:             kkhan@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+WildFly's (and WildFly Core's) testsuite runs at a particular stability level by default. At the time of writing this
+is set to the 'community' level.
+
+Since features may be introduced to WildFly at a lower stability level than this (e.g. 'experimental' and 'preview'), those features will not be available at the stability level the tests are currently run. Thus, we need a way to run tests at a chosen stability level.
+
+This proposal attempts to solve that by providing a set of ServerSetupTasks that will reload the server to the desired stability level before running the test. To support this a new management operation based on the existing `reload` operation will be introduced.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE[WFCORE-6728]
+
+=== Related Issues
+
+//* https://issues.redhat.com/browse/WFLY[WFLY-XXXX]
+
+=== Stability Level
+// Choose the planned stability level for the proposed functionality
+* [ ] Experimental
+
+* [ ] Preview
+
+* [x] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
```

Comment:
Please add you, me, Yeray (if he doesn't object), James.

This block (or a slightly tweaked variant) of the doc lists the feature team. Assignee (you) , SME(s) (me and Yeray), outside perspective person (James).

Reply:
@kabir I'm going to approve this, but please put yourself in here. :)

Reply:
That should be done as it's defined in the attributes.